### PR TITLE
First two sections of mobile website

### PR DESCRIPTION
**Pull request by BucurEva87**
Created the header and the hero section for the mobile template.

### CODE REVIEW

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @BucurEva87 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing successfully ✔️
The project is running successfully ✔️
Good commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	
+	<header id="main-header">
```
[the remark this answers is not in the document]
Comment (BucurEva87) [reply index.html:12]: I didn't really realize that that geometric shape was a logo. It was not clear to me from the design. I thought it was part of the design and that's precisely why I chose to make it a background image and not an image element. I will, however, wrap it in an anchor tag.

**Review by zhadier — APPROVED** [review]
## Hi @BucurEva87  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
- [ ] [_Optional_] Your Portfolio seems to have a fixed layout after 330px. It would help you out a lot if you make it fluid for now, so when you create the desktop version in the coming weeks  it won't be as much of a hassle as turning a fixed design into a responsive one. Some mobile devices have widths greater than 320 px so for those devices, your web app will just be centralized and not take up the entire screen as you can see below. You can use[ this resource](https://ux-alpaca.medium.com/so-what-exactly-is-the-difference-between-fixed-fluid-adaptive-and-responsive-layouts-and-why-3773272d8481) to get a better understanding but it's entirely optional and won't hinder your progress through the program so implementing the feature is entirely up to you
> 
image

<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  justify-content: space-around;
+}
+
+#hero h1 {
+  font: 5rem/100% 'Playfair Display', serif;
+}
+
+#hero p {
+  font: 1.4rem/2.1rem 'Poppins', sans-serif;
+  letter-spacing: -0.01em;
```
[the remark this answers is not in the document]
Comment (BucurEva87) [reply css/style.css:62]: Thank you! Yeah, I'm sort of lazy when it comes to CSS coding (actually I HATE CSS, 'cause I have no talent whatsoever when it comes to design, but I can implement layouts with it. There are a lot of shorthands that I know of - yet I'm almost never able to remember the order in which the properties are placed. I just remember which properties are covered by that specific shorthand :)) Thank you for your kind review and advice!

**Pull request merged**